packages/react-resizable-panels/src/vendor/react.ts

```diff
@@ -36,8 +36,8 @@ const {
   useState,
 } = React;
 
-// `toString()` prevents bundlers from trying to `import { useId } from 'react'`
-const useId = (React as any)["useId".toString()] as () => string;
+// `Math.random()` and `.slice(0, 5)` prevents bundlers from trying to `import { useId } from 'react'`
+const useId = (React as any)[`useId${Math.random()}`.slice(0, 5)] as () => string;
```

Comment:
I think the formatter wants this:
```suggestion
const useId = (React as any)[
  `useId${Math.random()}`.slice(0, 5)
] as () => string;
```